src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -25,19 +25,71 @@ const DomainList = (props) => {
     isHashtag,
     handleSetFollow,
     handleSetUnFollow,
+    handleSetBlock,
+    handleSetUnblock,
     onPressBody,
     DefaultImage,
-    isCommunity
+    isCommunity,
+    isBlockedSection
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new props `handleSetBlock`, `handleSetUnblock`, and `isBlockedSection` are introduced without any default values in the defaultProps. This could potentially lead to issues if these props are not provided when the component is used. It's a good practice to provide default values for all props to prevent unexpected behavior.
```diff
 DomainList.defaultProps = {
+  handleSetBlock: () => {},
+  handleSetUnblock: () => {},
+  isBlockedSection: false,
 };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->